config/application.rb

```diff
@@ -10,7 +10,10 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "action_cable/engine"
-# require "sprockets/railtie"
+
+require "sprockets/railtie"
```

Comment:
So this actually shouldnt be in here, the line was what was causing heroku to break. So it should be commented out.

Reply:
Should be updated now